tmail-backend/jmap/extensions-es/src/main/java/com/linagora/tmail/james/jmap/ESEmailAddressContactSearchEngine.java

```diff
@@ -44,12 +44,8 @@ public ESEmailAddressContactSearchEngine(ReactorElasticSearchClient client) {
     @Override
     public Publisher<Void> index(AccountId accountId, EmailAddressContact contact) {
         return Mono.fromRunnable(() -> {
-            String content = null;
-            try {
-                content = objMp.writeValueAsString(new AccountEmailContact(accountId, contact));
-            } catch (JsonProcessingException e) {
-                e.printStackTrace();
-            }
+            String content = Try.ofCallable(() -> objMp.writeValueAsString(new AccountEmailContact(accountId, contact)))
+                             .getOrElseGet(Throwable::toString);
```

Comment:
```
                             .getOrElseGet(Throwable::toString);
```

This do not qualify as valid error handling.

Please carry over the exceptions into the returned mono.